release-calendar/src/client/components/ChannelTable.tsx

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import '@material/react-list/index.scss';
+import * as React from 'react';
+import {Channel} from '../../types';
+import {DATA} from '../models/data';
+import {getCurrentReleases} from '../models/release-channel';
+import List, {
+  ListDivider,
+  ListGroup,
+  ListGroupSubheader,
+  ListItem,
+  ListItemText,
+} from '@material/react-list';
+
+export interface ChannelTableProps {
+  // TODO: considering moving the RTVs out of the ChannelTable Component altogether and
+  // intead having them live directly next door. Still testing.
+  handleSelectChannel: (selected: number) => void;
+  selectedChannel: number;
+}
+
+export interface ChannleTableState {
+  rtvs: Map<Channel, string>;
+}
+
+// eslint-disable-next-line prettier/prettier
+export class ChannelTable extends React.Component<ChannelTableProps, ChannleTableState> {
+  state: ChannleTableState;
+  constructor(props: Readonly<ChannelTableProps>) {
+    super(props);
+    this.state = {
+      rtvs: new Map(),
+    };
+    this.handleChannelChange = this.handleChannelChange.bind(this);
+  }
+
+  componentDidMount(): void {
+    Promise.resolve(getCurrentReleases(DATA)).then(result =>
+      this.setState({rtvs: result}),
+    );
+  }
+
+  handleChannelChange = (activatedItem: number): void => {
+    return activatedItem === this.props.selectedChannel
+      ? this.props.handleSelectChannel(null)
+      : this.props.handleSelectChannel(activatedItem);
+  };
+
+  render(): JSX.Element {
```

Comment:
It's unclear what the colors on the calendar mean. Can this list also serve as a legend, like in your UX mockups?